packages/main/src/types/BadgeDesign.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * Defines badge design types.
+ *
+ * @readonly
+ * @enum {string}
+ * @public
+ * @author SAP SE
+ * @alias sap.ui.webc.main.types.BadgeDesign
+ */
+enum BadgeDesign {
+	/**
+	 * Set1 of generic indication colors
```

Comment:
I suggest:
Set1 of generic indication colors that are intended only for industry-specific use cases